portal-api/src/v1/controllers/reports/review-ukef-decision.controller.js

```diff
@@ -17,12 +17,12 @@ const getUkefDecision = async (decision, bankId) => {
   //    dealType: "GEF"
   //    submissionDateEpoch: "1642762644833"
   // }]
-  const deals = await queryDb.aggregate([ // TODO SR-8
+  const deals = await queryDb.aggregate([
     { $unwind: '$ukefDecision' },
-    { $match: { 'ukefDecision.decision': decision, 'bank.id': bankId } },
+    { $match: { 'ukefDecision.decision': { $eq: decision }, 'bank.id': { $eq: bankId } } },
     {
       $project: {
-        _id: 0,
+        _id: false, // TODO SR-8 Changed values in project to true or false to better represent the functionality
```

Comment:
Remove todo once noted:
```suggestion
        _id: false,
```